textpipe/pipeline.py

```diff
@@ -1,20 +1,21 @@
 """
 Obtain elements from a textpipe doc, by specifying a pipeline, in a dictionary.
 """
+import json
 
 from textpipe.doc import Doc
-import json
+import textpipe.operation
 
 
 class Pipeline:
     """
     Create a pipeline instance based on the elements you would want from your text
 
-    >>> pipe = Pipeline(['raw', 'nwords', 'clean_text'])
+    >>> pipe = Pipeline(['Raw', 'Nwords', 'CleanText'])
     >>> sorted(pipe('Test sentence <a=>').items())
-    [('clean_text', 'Test sentence '), ('nwords', 2), ('raw', 'Test sentence <a=>')]
+    [('CleanText', 'Test sentence '), ('Nwords', 2), ('Raw', 'Test sentence <a=>')]
     """
-    def __init__(self, pipeline=None, **kwargs):
+    def __init__(self, operations, language=None, hint_language=None, **kwargs):
```

Comment:
There was some debate during our hackathon whether it is a good idea to give these language params to the pipeline, as opposed to the processors. I don't recall the exact opposition to the current way, was it you @anneschuth or @dodijk ?

Reply:
@anneschuth @dodijk, what is your opinion on this?

Reply:
I think indeed that they should be arguments to a processor (the language detection processor in this case). Other processors can then depend on the language detection processor if they care about language.

Reply:
I thing I agree. Language detection can just be another operation, but it will be one that many others depend on. As we do not have anything for operator parameters at the moment (@jinxcat was on this), I propose we move this to an issue and new PR.

Reply:
Indeed, but I need to wait for this to be merged to continue

Reply:
Created #39 for this, that is dependent on #10.